clio-server/src/main/scala/org/broadinstitute/clio/server/service/ReadGroupService.scala

```diff
@@ -49,23 +79,40 @@ object ReadGroupService {
   }
 
   private[service] val ConverterV1Key =
-    SameFieldsTypeConverter[TransferReadGroupV1Key, ModelReadGroupKey]
+    CaseClassTypeConverter[TransferReadGroupV1Key, ModelReadGroupKey] {
+      _.updated("location", "Unknown")
```

Comment:
I'm not saying we need a standard facility for handling fields that have been added to APIs (e.g. that lists the names and their default values) but having a constant means that the reader doesn't need to figure out that "location" is referring to the same thing everywhere it's used here. And if we had a test that verified that using v1 gives you "Unknown" for this API, you'd want a constant to avoid copying the value in the test.